src/rev_ai/models/transcript.py

```diff
@@ -36,16 +36,18 @@ def __init__(self, speaker, elements):
     def __eq__(self, other):
         """Override default equality operator"""
         if isinstance(other, self.__class__):
-            return self.__dict__ == other.__dict__
+            return all(a == b for a, b
+                       in zip(self.elements, other.elements)) \
+                   and self.speaker == other.speaker
         return False
 
     @classmethod
     def from_json(cls, json):
         """Alternate constructor used for parsing json"""
         return cls(
-            json["speaker"],
-            [Element.from_json(element) for element in json.get("elements", [])]
-        )
+            json['speaker'],
+            [Element.from_json(element)
+             for element in json.get('elements', [])])
```

Comment:
just curious why is this line wrapped

Reply:
It hits the 80 character limit.

Reply:
80 character limit does seem really crazy in this case

Reply:
Will be set to 100